### PR TITLE
remove old grading function implementations

### PR DESCRIPTION
**Pull request by rogthefrog**

### CODE REVIEW

File: src/modelgauge/main.py [hunk 1]
```diff
@@ -207,7 +200,7 @@ def run_test(
     secrets = load_secrets_from_config()
     # Check for missing secrets without instantiating any objects
     test_cls = TESTS._get_entry(test).cls
-    annotators = test_cls.get_annotators()
+    annotators = test_cls.get_annotators(test_cls)
```
Comment (bkorycki) [inline src/modelgauge/main.py:203]: why add this arg? I think this is the cause of some of the failing tests

File: plugins/mistral/tests/test_mistral_sut.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def response():
 
 @pytest.fixture
 def sut():
-    return MistralAISut("ministral-8b-latest", "ministral-8b-latest", "latest", MistralAIAPIKey("fake"))
+    return MistralAISut("ministral-8b-latest", "ministral-8b-latest", MistralAIAPIKey("fake"))
```
Comment (bkorycki) [inline plugins/mistral/tests/test_mistral_sut.py:44]: Same here.

File: src/modelgauge/main.py [hunk 3]
```diff
@@ -1,40 +1,33 @@
...
 import pathlib
 import warnings
 from typing import List, Optional
 
+import click
+
 from modelgauge.annotator import CompletionAnnotator
 from modelgauge.annotator_registry import ANNOTATORS
 from modelgauge.base_test import PromptResponseTest
 from modelgauge.command_line import (
```
Comment (bkorycki) [inline src/modelgauge/main.py:12]: I kind of like it when constants are grouped together at the beginning of an import. This is just a preference though,

**Review by bkorycki — APPROVED** [review]
Looks good! thanks

File: src/modelgauge/main.py (hunk 3 above)
Comment (rogthefrog) [reply src/modelgauge/main.py:12]: The changes here are done automatically by the standard rules in the Black linter. I can tweak them if you'd like.

Comment (bkorycki) [reply src/modelgauge/main.py:12]: Hm, I also use black but it doesn't format stuff this way for me. Maybe we have different configurations?

**Pull request merged**